README.md

```diff
@@ -1,72 +1,72 @@
+- **Ethernet** - Ethernet MAC and PHY peripheral interface driver.
+- **I2C** - Inter-Integrated Circuit multi-master serial single-ended bus interface - driver.
+- **MCI** - Memory Card Interface driver for SD/MMC memory.
+- **SPI** - Serial Peripheral Interface driver.
+- **USART** - Universal Synchronous and Asynchronous Receiver/Transmitter interface driver.
+- **USB Device** - Universal Serial Bus Device interface driver.
+- **USB Host** - Universal Serial Bus Host interface driver.
+- **WiFi** - Wireless Fidelity Interface module/shield driver.
 
-### Issues and Labels
+## Repository Structure
 
-Please feel free to raise an [issue on GitHub](https://github.com/ARM-software/CMSIS-Driver_Validation/issues)
-to report misbehavior (i.e. bugs) or start discussions about enhancements. This
-is your best way to interact directly with the maintenance team and the community.
-We encourage you to append implementation suggestions as this helps to decrease the
-workload of the very limited maintenance team. 
+| Directory/File        | Content                                                   |
+| --------------------- | --------------------------------------------------------- |
+| `./Boards/`           | Driver Validation examples for various boards             |
+| `./Config/`           | Configuration files for the driver validation framework   |
+| `./Documentation/`    | Placeholder for the offline documentation in the pack     |
+| `./DoxyGen/`          | Source of the documentation                               |
+| `./Include/`          | Header files for driver validation components             |
+| `./Scripts/`          | Script files for XML reports                              |
+| `./Source/`           | Source files for driver validation components             |
+| `./Tools/`            | Server side tools for testing drivers                     |
+| `./ARM.CMSIS-Driver_Validation.pdsc` | Open-CMSIS-Pack description file           |
+| `./gen_pack.sh`       | Open-CMSIS-Pack generation script                         |
+| `./LICENSE.txt`       | License text for the repository content                   |
 
-We will be monitoring and responding to issues as best we can.
-Please attempt to avoid filing duplicates of open or closed items when possible.
-In the spirit of openness we will be tagging issues with the following:
 
-- **bug** – We consider this issue to be a bug that will be investigated.
 
-- **wontfix** - We appreciate this issue but decided not to change the current behavior.
+## Examples
 
-- **enhancement** – Denotes something that will be implemented soon. 
+Folder [./Boards/](./Boards/) contains example projects that show how to use the CMSIS-Driver Validation on a real hardware with available CMSIS-Driver implementations.The examples are also included in CMSIS-Driver Validation Software Pack. Details are explained in [Examples documentation](https://arm-software.github.io/CMSIS-Driver_Validation/main/examples.html).
```

Comment:
I would rename './Boards/' to just 'Boards'

Reply:
Agree. Will update